files/alembic.ini

```diff
@@ -0,0 +1,74 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = /usr/share/securedrop-client/alembic
```

Comment:
If we're going to be running alembic, we shouldn't be using a hard coded value you like this. We should use a template and a `%%ALEMBIC_PATH%%` or something similar to replace the values in a pre-`alembic` step.

Reply:
This is to help the Debian packaging. All packaged applications (as far as I know) in Fedora land also uses the same idea.

Reply:
Right, but the `alembic.ini` is useless without this line

```ini
sqlalchemy.url = sqlite:////path/to/database.sqlite
```

The app at boot *could* pull an `alembic.ini.tmpl` from `/var/lib/` or `/usr/share` and then inject that value before running `alembic upgrade head`.

Reply:
It is here for the default path in the Debian package https://github.com/freedomofpress/securedrop-client/pull/86/files#diff-3f5cd7c244dcadbe672e3644a1f283bdR38